examples/multi-services/back-svc/main.go

```diff
@@ -25,13 +26,22 @@ func main() {
 	if err != nil {
 		log.Fatalf("unable to initialize tracer provider due: %v", err)
 	}
+
+	// initialize metrics middleware
+	metricsMiddleware, err := utils.NewMetricProvider(serviceName)
+	if err != nil {
+		log.Fatalf("unable to initialize metrics due: %v", err)
+	}
+
 	// define router
 	r := chi.NewRouter()
 	r.Use(otelchi.Middleware(serviceName, otelchi.WithChiRoutes(r)))
+	r.Use(metricsMiddleware)
 	r.Get("/", utils.HealthCheckHandler)
 	r.Get("/name", func(w http.ResponseWriter, r *http.Request) {
 		w.Write([]byte(generateName(r.Context(), tracer)))
 	})
+	r.Handle("/metrics", promhttp.Handler())
```

Comment:
I think we can just send metrics data into prometheus server without being scraping it into our server. You may need to check on [this](https://prometheus.io/docs/guides/opentelemetry/) or if it doesnt works well consider of using otel-collector, the example can be seen [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/examples/demo/docker-compose.yaml)

Reply:
hey @ilhamsyahids, thanks for your feedback. i have resolve your request about metrics using otel-collector, tested localy and it's working. please let me know if there's any request that you need to confirm